jiant/preprocess.py

```diff
@@ -661,7 +663,9 @@ def add_task_label_vocab(vocab, task):
         return
     log.info("\tTask '%s': adding vocab namespace '%s'", task.name, namespace)
 
-    if isinstance(task, SequenceGenerationTask):
+    if isinstance(task, SequenceGenerationTask) and not isinstance(
+        task, MaskedLanguageModelingTask
```

Comment:
Looking for a comment here (in the code) to explain why this special case/handling is added.

Reply:
I think his was from some artifact old code from when LM was first put into the repo and test on non-transformer models. I'm actually going to remove this block, since our Transformer models add special tokens elsewhere to the vocab.